LayoutTests/fast/forms/auto-fill-button/input-loading-auto-fill-button.html

```diff
@@ -0,0 +1,16 @@
+<!DOCTYPE html>
+<html>
+<body>
+<p>This tests that the Loading AutoFill button renders. It can only be tested in the test tool.</p>
+<div id="container">
+    <input type="text">
+</div>
+<script>
+
+if (window.internals) {
+    const input = document.querySelector("input");
+    window.internals.setShowAutoFillButton(input, "Loading");
+}
```

Comment:
NIT: you could rewrite this as `window.internals?.setShowAutoFillButton(input, "Loading")`

Reply:
oh, null coalescing is in JS too? I thought it was just a TypeScript thing. I'll change it

Reply:
this is optional chaining actually 🙂

both optional chaining and nullish coalescing are JS features, not TypeScript features 😉